src/sdk/namespace_fs.js

```diff
@@ -1426,6 +1426,15 @@ class NamespaceFS {
         }
     }
 
+    _is_force_md5_enabled(object_sdk) {
```

Comment:
Now that we know the use, I think that we need to have the option to "unset" it (to make it `undefined`) using the CLI. WDYT?
The scenario: bucket was added with the `--force_md5_etag true` by mistake, and one wishes to use the defaults like other buckets, how can it be done? (changing it to `false` back might not be aligned when the config changes).

Reply:
I think it sounds good, this part 

> "maybe we should set it to be a string and allow only true/false and '' value to unset it"


will be only for this flag, right? because in other boolean flags, I'm not sure we have (or want) the option to unset them.